app/models/client.rb

```diff
@@ -192,6 +192,12 @@ def clients_with_dupe_contact_info
     Client.after_consent.where(intake: matching_intakes).pluck(:id)
   end
 
+  # We'll consider a client to have completed intake process once any of their
+  # tax returns has passed the intake stage
+  def completed_intake_process?
```

Comment:
now it seems to me like we are using 3 different things to determine where the client is:
- the last page they were on (`current_step`)
- their question answers (`determine_current_question`)
- the status of their tax returns (this method)

Reply:
current_step is the current step when they go through the process. Determine_current_question provides the backfill to determine the current_step for clients who went through the flow _before_ we started updating current_step on every page. This method is what determines whether we care about that _at all_ because once your tax returns are beyond a certain status we don't send you back into the "onboarding flow"